pkgs/tools/networking/slurm-cli/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkg-config
+, meson
+, ncurses
+, ninja
+}:
+
+stdenv.mkDerivation rec {
+  pname = "slurm";
+  version = "0.4.4";
+
+  src = fetchFromGitHub {
+    owner = "mattthias";
+    repo = "slurm";
+    rev = "v${version}";
+    hash = "sha256-w77SIXFctMwwNw9cQm0HQaEaMs/5NXQjn1LpvkpCCB8=";
+  };
+
+  nativeBuildInputs = [ pkg-config meson ninja ];
+  buildInputs = [ ncurses ];
+
+  intallPhase = ''
+    mkdir -p "$out/bin"
+  '';
+
+  meta = with lib; {
+    description = "A generic network load monitor";
+    homepage = "https://github.com/mattthias/slurm";
+    license = licenses.gpl2;
```

gpl2 is deprecated, always use gpl2Plus or gpl2Only depending on the repo licensing.
in this case, they explicitly allow "any later version" in the source code headers (e.g. look at slurm.c), so:
```suggestion
    license = licenses.gpl2Plus;
```